### DISCUSSION TITLE
Demo source code?

### DISCUSSION THREAD

**Discussion by amirasyraf** [area17/twill#869] [Q&A]
Hi there,

Where could I find the source code repo for the demo at https://demo.twill.io/ ? 

Thanks

**Comment by SH-A-DOW** [comment]
not going to happen, secrecy for some reason.

**Comment by amirasyraf** [comment]
> not going to happen, secrecy for some reason.

Is that something they've addressed before?

**Comment by SH-A-DOW** [comment]
> > not going to happen, secrecy for some reason.
> 
> Is that something they've addressed before?

Demo source has been requested by users since 2019, they said they will make better tutorials, you work it out.

**Comment by amirasyraf** [comment]
Well, that's unfortunate. Thanks. I guess I'll have to rely on docs. 

Just kidding, the docs sucks, you'd have to dig through the source code to do anything remotely useful.

**Comment by amirasyraf** [comment]
On second thought, I'll just leave this issue open until we receive an official response.

**Comment by mrdoinel** [comment]
There are some videos Tutorials with source codes to create a simple CMS.
This will probably help :
https://www.youtube.com/channel/UC5rhdsVLs4vPVuy_dAaRzhA

**Reply by refaelgold** [reply]
Its not relevant anymore because they change some stuff

**Comment by refaelgold** [comment]
Totally agreed.
If Twill has hard coded Analytics or something just remove it and share the source code, please. 

Someone can tag them? 
You guys have maybe repo's code for advanced cms?

**Reply by pboivin** [reply]
Hi @refaelgold! Have a look here: https://github.com/area17/awesome-twill#open-source-twill-projects
There are a few examples of open source, real production sites using Twill.
